### PR TITLE
Config Migration

### PR DESCRIPTION
**Pull request by felire**
## Summary

Config migration

## Trello Card

[https://trello.com/c/mItj3qMa](https://trello.com/c/mItj3qMa)

### CODE REVIEW

File: generators/app/templates/src/config/api.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ const api = create({
   timeout: 5000
 });
 
-api.addMonitor(Reactotron.apisauce);
+api.addMonitor(((Reactotron as unknown) as { apisauce: any }).apisauce);
```
Comment (sfernandez11) [inline generators/app/templates/src/config/api.ts:16]: :magic: :que-trucazo: hahaha

File: generators/app/templates/src/config/navigation.ejs [hunk 2]
```diff
@@ -7,11 +8,11 @@ import { DrawerActions } from 'react-navigation-drawer';
 import Routes from '@constants/routes';
 import { blue, white } from '@constants/colors';
 import statusBarConfig from '@constants/statusBar';
-
+import { <%_ if (features.tabs) { _%> AppTabOptions,<%_ } _%> Navigation, AppScreensNavOptions } from '@interfaces/navigation';
 import fonts from './fonts';
```
Comment (sfernandez11) [inline generators/app/templates/src/config/navigation.ejs:12]: Missing line break: 
```
import statusBarConfig from '@constants/statusBar';
import { <%_ if (features.tabs) { _%> AppTabOptions,<%_ } _%> Navigation, AppScreensNavOptions } from '@interfaces/navigation';

import fonts from './fonts';

// Default nav options for all screens
...
```

File: generators/app/templates/src/config/reactotronConfig.ejs [hunk 3]
```diff
@@ -3,45 +3,41 @@ import Reactotron, { asyncStorage, overlay, trackGlobalErrors } from 'reactotron
...
   const scriptHostname = scriptURL.split('://')[1].split(':')[0];
   Reactotron.configure({ name: '<%= projectName %>', host: scriptHostname })
-    .use(trackGlobalErrors())
+    .use(trackGlobalErrors({}))
     .use(apisaucePlugin())
     .use(
       reactotronRedux({
         onRestore: state =>
           Object.entries(state).reduce(
+            // eslint-disable-next-line new-cap
```
Comment (sfernandez11) [inline generators/app/templates/src/config/reactotronConfig.ejs:25]: I think this  `eslint-disable` is not necessary. The rule is already deactivated in the [.eslintrc](https://github.com/Wolox/wolmo-bootstrap-react-native/blob/master/generators/app/templates/.eslintrc.ejs) file

File: generators/app/templates/src/config/api.ts (hunk 1 above)
Comment (felire) [reply generators/app/templates/src/config/api.ts:16]: I was having a war with this, it was impossible to type this on a nice way, if you know I will be very glad for some help haha

**Pull request merged**